src/autofill-utils.js

```diff
@@ -397,13 +402,30 @@ const wasAutofilledByChrome = (input) => {
 }
 
 /**
- * Checks if we should log debug info to the console
+ * Checks if we should log form analysis debug info to the console
  * @returns {boolean}
  */
 function shouldLog () {
+    return readDebugSetting('ddg-autofill-debug')
+}
+
+/**
+ * Checks if we should log performance info to the console
+ * @returns {boolean}
+ */
+function shouldLogPerformance () {
+    return readDebugSetting('ddg-autofill-perf')
```

Comment:
just to be 100% sure on this - we never *write* these values to storage in code do we? It's purely a manual step that you can take (opening dev tools and inserting the value)